modules/enterprise/server/jar/src/main/java/org/rhq/enterprise/server/scheduler/jobs/DataPurgeJob.java

```diff
@@ -145,6 +146,12 @@ private void purgeEverything(SystemSettings systemSettings) {
         purgeResourceConfigHistory(systemSettings);
         removeResourceErrorDuplicates();
         removeStaleAvailabilityResourceErrors();
+        purgeOldPackageBits();
+    }
+
+    private void purgeOldPackageBits(){
+        ContentManagerLocal cm = LookupUtil.getContentManager();
```

Comment:
Rest of the class does this lookup in the DataPurgeJob instantiation, this should be there also for the clarity.